### ISSUE TITLE
3rd level menu item resets all the menu

### ISSUE THREAD

**Issue by gcolotti** [thedevdojo/voyager#3806]
## Version information

 - Laravel: v5.7.*
 - Voyager: v1.1.*
 - PHP: 7.1
 - Database: MySQL

## Description
When any 3rd level menu item is clicked, the expanded menus are reseted.

## Steps To Reproduce
1. Build a 3rd level menu item
2. Click on it.

## Expected behavior
When a 3rd level menu item is clicked, keep the menu expanded.

**Comment by emptynick** [comment]
Please search before opening new issues.
Closing as a duplicate of https://github.com/the-control-group/voyager/issues/2125
Will be fixed in https://github.com/the-control-group/voyager/pull/3620

**Comment by gcolotti** [comment]
No, is not the same. I will explain better.

In #3620, ta-tikoma fixed #2125 only, which doesn't even let you use a 3rd lvl menu. Also, in this [comment](https://github.com/the-control-group/voyager/issues/2125#issuecomment-377687142) in the same post, @f0reclone added the missing CSS for #3620.

I added to custom JS and the CSS previously mentioned to custom js and css files, and 3rd menu items are now working like a charm. But there is still an issue: when you click in a 3rd lvl menu item (with 3rd lvl menu WORKING), when the sidebar is refreshed (to get the content of the section of the desired menu item, i.e. voyager.posts.index), the expanded items are collapsed. This do not happen to 2nd lvl items: when click a 2nd lvl item and the sidebar is refreshed after getting the section content, the 2nd lvl items are still expanded. First I thought it was due to the 'active' class in 'li' elements, but is not.
Hope it is clear enought now.
